src/charts/bar.js

```diff
@@ -961,7 +986,7 @@ define(function(require) {
         /**
          * Gets or Sets whether the color list should be reversed or not
          * @param  {boolean} _x     Should reverse the color list
-         * @return { boolean | module} Is color list being reversed
+         * @return {boolean | module} Is color list being reversed
```

Comment:
Could add "or chart module to chain calls" to keep in line with return documentation for most properties